lib/jake/number.ex

```diff
@@ -1,5 +1,157 @@
 defmodule Jake.Number do
-  def gen(_spec) do
-    StreamData.integer()
+  @num_min 0
+
+  @num_max 100_000
```

Comment:
why is the default so low? The generator should ideally generate all possible value, assuming the cost of generation is not that high.

Reply:
They are just temporary, I also wanted to discuss about possible limits. Cost of generation is not high, except for floating point multiple, which I'll look into afterwards.